backend/src/main/resources/application.properties

```diff
@@ -11,6 +11,11 @@ TOKEN_DB_PASSWORD=cassandra
 TOKEN_DB_CLUSTER=Test Cluster
 TOKEN_DB_LOCAL_DATACENTER=datacenter1
 TOKEN_DB_KEYSPACE=tokenkeyspace
+JWT_SECRET=jfgi95jgnfi4
+ACCESS_EXPIRATION_MS=1800000
+EMAIL_EXPIRATION_MS=1800000
+SPRING_MAIL_USERNAME=no.reply.skynest@gmail.com
+SPRING_MAIL_PASSWORD=ftzldfejzhtqjsok
```

Comment:
This values I believe should be set in .env file, by this I mean: JWT_SECRET and SPRING_MAIL_PASSWORD. Others can be left hard coded as they previously were.

The point of using this .env variables is to extract the sensitive information in separate place so they will not be attached with the code, but there reference will be used instead.